fission/src/systems/simulation/wpilib_brain/WPILibBrain.ts

```diff
@@ -1,24 +1,35 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Not sure if you meant to delete this or not but the linter is certainly not happy about it.

Reply:
Fixed